### ISSUE TITLE
Catalog Price Rules - add duplicate rule action

### ISSUE THREAD

**Issue by seansan** [OpenMage/magento-lts#488]
Feature request

Catalog Price Rules - add duplicate rule action in grid

**Comment by seansan** [comment]
I dont see your point ...  the example is not even about  *catalog* price
rules

The request is simple: action to duplicate *catalog* price rules

Helps us and many others out a lot. And it is only a request. You can reset
title on duplication to force saving as a new rule.

**Comment by sreichel** [comment]
I think you can add massactions (or a duplicate button) via observer.

IMHO this should not be added.

**Comment by SaintLuticriss** [comment]
Duplicating Coupons/Promotions/Rules should be a native function. 
Remember the audience of users you're trying to code for: they're "business degree" doers; not "computer science degree" tinkerers.
They don't want to spend time developing the platform, they want to develop their business.
I find it regrettable that in order to get this program working as expected by the common business owner, I have to install a dozen 50-line "extensions" every single time. That makes them more like "fixes" than extensions though, eh?

**Comment by sreichel** [comment]
@SaintLuticriss from "business viewpoint" I totally agree, but then we had to add a plenty of new features to make everybody happy. 

**IMHO** _"the audience of users we're trying to code for"_ are techies. As "normal business guy" you dont know or care about such forks. I guess all maintainers are either developers are have an affinity for magento.

This project should only cover bugs, performance issues and small core features that are can not be realized without rewrites. Adding grid colums/mass actions are more or less simple features that you implement "day by day". 

I prefer to install a bunch of "50-line-extension" over having a "feature-messed-up" install that cause more problems. (just check M2 issues related to composer install that includes 3rd-party-plugins - that are not really tested by M2 team.)

Just to mention ... its just a few people who maintain this. Every features takes time to test , time to cover possible problems with M1 patches.

TLDR: if possible w/o rewrites it does not belong here. (IMHO)
